dotnet/src/Agents/Abstractions/AgentChat.cs

```diff
@@ -14,14 +13,31 @@ namespace Microsoft.SemanticKernel.Agents;
 /// <summary>
 /// Point of interaction for one or more agents.
 /// </summary>
+/// <remarks>
+/// Any <see cref="AgentChat" /> instance does not support concurrent invocation and
+/// will throw exception if concurrent activity is attempted for:
+/// - <see cref="AddChatMessage"/>
+/// - <see cref="AddChatMessages"/>
+/// - <see cref="GetChatMessagesAsync"/>
+/// - <see cref="InvokeAgentAsync"/>
+/// </remarks>
 public abstract class AgentChat
 {
     private readonly BroadcastQueue _broadcastQueue;
-    private readonly Dictionary<string, AgentChannel> _agentChannels;
-    private readonly Dictionary<Agent, string> _channelMap;
+    private readonly Dictionary<string, AgentChannel> _agentChannels; // Map channel hash to channel: one entry per channel.
+    private readonly Dictionary<Agent, string> _channelMap; // Map agent to its channel-hash: one entry per agent.
 
     private int _isActive;
 
+    /// <summary>
+    /// Indicates if a chat operation is active.  This includes:
+    /// - <see cref="AddChatMessage"/>
+    /// - <see cref="AddChatMessages"/>
+    /// - <see cref="GetChatMessagesAsync"/>
+    /// - <see cref="InvokeAgentAsync"/>
```

Comment:
Not getting the significance of all the class references, can you expand the comment a bit?

Reply:
Intent was to call out the public methods on this class.  I agree (with sync dialog) that being this explicit is just a point of maintenance.  Updated as suggested.